flang/lib/Optimizer/Transforms/RewriteLoop.cpp

```diff
@@ -147,31 +146,30 @@ class CfgIfConv : public mlir::OpRewritePattern<fir::WhereOp> {
     auto opPosition = rewriter.getInsertionPoint();
     auto *remainingOpsBlock = rewriter.splitBlock(condBlock, opPosition);
     mlir::Block *continueBlock;
-    if (where.getNumResults() == 0) {
+    if (ifop.getNumResults() == 0) {
```

Comment:
This probably should be `ifOp`.

Reply:
Thanks for pointing this out, must've missed out.
Done changes!